openslides_backend/action/actions/agenda_item/agenda_creation.py

```diff
@@ -56,18 +56,23 @@ def check_dependant_action_execution_agenda_item(
         )
         agenda_item_creation = meeting.get("agenda_item_creation")
         agenda_create = instance.pop("agenda_create", None)
+        result_value: Optional[bool] = None
```

Comment:
Shouldn't you be able to use 
```suggestion
        result_value: bool
```
since it is always assigned a value in the if-else? Then you can also drop the `cast` below

Reply:
done